charts/k8ssandra/values.yaml

```diff
@@ -565,13 +571,16 @@ stargate:
   tolerations: []
 reaper:
   # -- Security context override for reaper container
-  securityContext: {}
+  securityContext:
+    runAsNonRoot: true
   # -- Security context override for reaper schema init container
   schemaInitContainerConfig:
-    securityContext: {}
+    securityContext:
+      runAsNonRoot: true
```

Comment:
We also need to specify `runAsUser` here; otherwise, we will get this error:

```yaml
        message: container has runAsNonRoot and image has non-numeric user (reaper),
          cannot verify user is non-root
        reason: CreateContainerConfigError
```